docs/changelog.rst

```diff
@@ -1,6 +1,14 @@
 ChangeLog
 =========
 
+3.0.0 (unreleased)
+------------------
+
+*New:*
+
+    - :issue:`316`: :class:`~factory.django.DjangoModelFactory` no longer calls ``.save()`` after `postgeneration` hooks. This is a **breaking change**. Call ``.save()`` in custom `postgeneration` hooks to maintain the previous behavior.
```

Comment:
Like was done for release 3.0.0, I think there should be a "Breaking changes" header to better highlight this to users. I also think it could further emphasize that users must take action (normally) as the change could be subtle in some scenarios for some users.

Reply:
I agree the change is very likely to cause breakage for users, possibly in subtle ways, and should be made very visible. I created a “Breaking changes” section in the latest revision.